src/core/renderDOM.ts

```diff
@@ -0,0 +1,10 @@
+import Block from './Block';
+
+export default function renderDOM(BlockPage: typeof Block) {
+  const block = new BlockPage();
+
+  const root = document.querySelector('#app');
+  
+  root!.innerHTML = '';
+  root!.appendChild(block.getContent());
```

Comment:
Стоит избегать `!` - он не спасает от ошибки в рантайме. Лучше использовать `?`
```suggestion
  root?.innerHTML = '';
  root?.appendChild(block.getContent());
```

Reply:
Здесь ошибка ts
The left-hand side of an assignment expression may not be an optional property access.ts(2779)

Reply:
А да, в 1 случае (с присваиванием) это не работает. Тогда лучше написать if